ui/tests/integration/components/masked-input-test.js

```diff
@@ -125,34 +125,4 @@ module('Integration | Component | masked input', function (hooks) {
       .dom('[data-test-icon="minus"]')
       .exists('shows minus icon when unmasked because value is empty string');
   });
-
-  test('it shows "success" flash message when the value is successfully copied', async function (assert) {
```

Comment:
Removed these 2 tests because there is no flash message with the HDS copy button